.github/workflows/build.yml

```
@@ -16,10 +16,10 @@ jobs:
             - name: Checkout repository
               uses: actions/checkout@v3
 
-            - name: Use Node.js 18
+            - name: Use Node.js 22
               uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)